client/my-sites/plugins/plugin-management-v2/plugin-action-status/render-status-message.tsx

```diff
@@ -0,0 +1,180 @@
+			[ PLUGIN_INSTALLATION_IN_PROGRESS ]: selectedSite
+				? translate( 'Deactivating' )
+				: translate(
+						'Deactivating on %(count)s site',
+						'Deactivating on %(count)s sites',
+						translationArgs
+				  ),
+			[ PLUGIN_INSTALLATION_COMPLETED ]: selectedSite
+				? translate( 'Deactivated' )
+				: translate(
+						'Deactivated on %(count)s site',
+						'Deactivated on %(count)s sites',
+						translationArgs
+				  ),
+			[ PLUGIN_INSTALLATION_ERROR ]: selectedSite
+				? translate( 'Failed to deactivate' )
+				: failedStatusMessage(
+						translate(
+							'Failed to deactivate on %(count)s site',
+							'Failed to deactivate on %(count)s sites',
+							translationArgs
+						)
+				  ),
+		},
+		[ ENABLE_AUTOUPDATE_PLUGIN ]: {
+			[ PLUGIN_INSTALLATION_IN_PROGRESS ]: selectedSite
+				? translate( 'Enabling auto-updates' )
+				: translate(
+						'Enabling auto-updates on %(count)s site',
+						'Enabling auto-updates on %(count)s sites',
+						translationArgs
+				  ),
+			[ PLUGIN_INSTALLATION_COMPLETED ]: selectedSite
+				? translate( 'Auto-update enabled' )
+				: translate(
+						'Auto-update enabled on %(count)s site',
+						'Auto-update enabled on %(count)s sites',
+						translationArgs
+				  ),
+			[ PLUGIN_INSTALLATION_ERROR ]: selectedSite
+				? translate( 'Failed to enable auto-updates' )
+				: failedStatusMessage(
+						translate(
+							'Failed to enable auto-updates on %(count)s site',
+							'Failed to enable auto-updates on %(count)s sites',
+							translationArgs
+						)
+				  ),
+		},
+		[ DISABLE_AUTOUPDATE_PLUGIN ]: {
+			[ PLUGIN_INSTALLATION_IN_PROGRESS ]: selectedSite
+				? translate( 'Disabling auto-updates' )
+				: translate(
+						'Disabling auto-updates on %(count)s site',
+						'Disabling auto-updates on %(count)s sites',
+						translationArgs
+				  ),
+			[ PLUGIN_INSTALLATION_COMPLETED ]: selectedSite
+				? translate( 'Auto-update disabled' )
+				: translate(
+						'Auto-update disabled on %(count)s site',
+						'Auto-update disabled on %(count)s sites',
+						translationArgs
+				  ),
+			[ PLUGIN_INSTALLATION_ERROR ]: selectedSite
+				? translate( 'Failed to disable auto-updates' )
+				: failedStatusMessage(
+						translate(
+							'Failed to disable auto-updates on %(count)s site',
+							'Failed to disable auto-updates on %(count)s sites',
+							translationArgs
+						)
+				  ),
+		},
+		[ REMOVE_PLUGIN ]: {
+			[ PLUGIN_INSTALLATION_IN_PROGRESS ]: selectedSite
+				? translate( 'Removing Plugin' )
+				: translate(
+						'Removing plugin from %(count)s site',
+						'Removing plugin from %(count)s sites',
+						translationArgs
+				  ),
+			[ PLUGIN_INSTALLATION_ERROR ]: selectedSite
+				? translate( 'Failed to remove plugin' )
+				: translate(
+						'Failed to remove plugin from %(count)s site',
+						'Failed to remove plugin from %(count)s sites',
+						translationArgs
+				  ),
+		},
+	};
+};
+
+export default function RenderStatusMessage( {
```

Comment:
I think the messages in that component are too long to fit in here, I guess we are covering all the cases(all actions with all statuses) currently.

Reply:
What do you think, @grbicsanja?

Reply:
I think we cover all the cases - all actions and all statuses - in a more detailed way I'd say (ie for messages like 'updated 5 plugins', we actually show which ones, and give a lot more valuable information than the global notice). I think it's safe to remove those.

In this list I see some others like:

`additionalExplanation = ( error_code ) => {
		const { translate } = this.props;

		switch ( error_code ) {
			case 'no_package':
				return translate( "Plugin doesn't exist in the plugin repo." );

			case 'resource_not_found':
				return translate( 'The site could not be reached.' );

			case 'download_failed':
				return translate( 'Download failed.' );

			case 'plugin_already_installed':
				return translate( 'Plugin is already installed.' );

			case 'incompatible_archive':
				return translate( 'Incompatible Archive. The package could not be installed.' );

			case 'empty_archive_pclzip':
				return translate( 'Empty archive.' );

			case 'disk_full_unzip_file':
				return translate( 'Could not copy files. You may have run out of disk space.' );

			case 'mkdir_failed_ziparchive':
			case 'mkdir_failed_pclzip':
				return translate( 'Could not create directory.' );

			case 'copy_failed_pclzip':
				return translate( 'Could not copy file.' );

			case 'md5_mismatch':
				return translate( "The checksum of the files don't match." );

			case 'bad_request':
				return translate( 'Invalid Data provided.' );

			case 'fs_unavailable':
				return translate( 'Could not access filesystem.' );

			case 'fs_error':
				return translate( 'Filesystem error.' );

			case 'fs_no_root_dir':
				return translate( 'Unable to locate WordPress Root directory.' );

			case 'fs_no_content_dir':
				return translate( 'Unable to locate WordPress Content directory (wp-content).' );

			case 'fs_no_plugins_dir':
				return translate( 'Unable to locate WordPress Plugin directory.' );

			case 'fs_no_folder':
				return translate( 'Unable to locate needed folder.' );

			case 'no_files':
				return translate( 'The package contains no files.' );

			case 'folder_exists':
				return translate( 'Destination folder already exists.' );

			case 'mkdir_failed':
				return translate( 'Could not create directory.' );

			case 'files_not_writable':
				return translate(
					'The update cannot be installed because we will be unable to copy some files. This is usually due to inconsistent file permissions.'
				);`

These seem to be related to installing and relevant on the plugin detail page?  We could keep these as global notices. Wdyt?

Are there any other specific messages in this list you noticed that we would lose but could be important? I could look into them and try and adjust the wording to make them shorter.

Reply:
I added a new card and we can explore the opportunities here post-release: 1202518759611394-as-1202919388468349